torch/nn/functional.py

```diff
@@ -62,8 +62,8 @@
 
 Examples::
 
-    >>> filters = torch.randn(33, 16, 3)
-    >>> inputs = torch.randn(20, 16, 50)
+    >>> inputs = torch.randn(33,16,30)
```

Comment:
Could you add back the proper spacing after the `,` please?

Reply:
@albanD  Thanks for reply,modification done